src/utils/auto_settings.py

```diff
@@ -44,10 +44,13 @@ def backup_settings():
             return
     f = open(current_file)
     curr_settings = json.load(f)
-    launch_options = curr_settings["Games"]["osi"]["AdditionalLaunchArguments"]
-    with open(backup_file, 'w') as f:
-        f.write(launch_options)
-    print("D2R launch options backed up successfully.")
+    try:
+        launch_options = curr_settings["Games"]["osi"]["AdditionalLaunchArguments"]
+        with open(backup_file, 'w') as f:
+            f.write(launch_options)
+        print("D2R launch options backed up successfully.")
+    except KeyError:
+        pass
```

Comment:
Put a Logger.warning saying that botty failed to set the launch options and possibly tell the user how to solve the issue, other than that looks good :D